### PR TITLE
Showers clean contamination and make sound

### PR DESCRIPTION
**Pull request by ninjanomnom**
:cl: ninjanomnom
add: Objects placed under showers are cleansed of radioactive contamination over a short time.
soundadd: Showers make sound now.
/:cl:

(I was going to rewrite showers but godamn that's actualy a major project and I have more than enough on my plate atm)

I also made a looping sound datum for future use

### CODE REVIEW

File: code/datums/looping_sounds/looping_sound.dm [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/datum/looping_sound
```
Comment (RemieRichards) [inline code/datums/looping_sounds/looping_sound.dm]: huge overkill, just set ```repeat``` on a ```/sound```

Comment (ninjanomnom) [reply code/datums/looping_sounds/looping_sound.dm]: wait hold on, I asked fucking 4 times if looping sounds we a possible thing and no one said this?

Regardless, this doesn't work when the loop consists of random sounds

Comment (RemieRichards) [reply code/datums/looping_sounds/looping_sound.dm]: well then that's a ```/datum/looping_RANDOM_sound``` then :^)
and of course nobody said, now I'm gone a huge chunk of knowledge has come with me :3

Comment (ninjanomnom) [reply code/datums/looping_sounds/looping_sound.dm]: it's not just random though, this also supports sounds that happen before and after the loop starts so you can have a transition.

Comment (RemieRichards) [reply code/datums/looping_sounds/looping_sound.dm]: well that's just 3 sounds.
you've already validated why the datum exists, you can stop trying now.

Comment (ninjanomnom) [reply code/datums/looping_sounds/looping_sound.dm]: also making these kinds of sounds their own datums makes it easier to change machinery droning sounds etc across the whole code at once.

**Pull request merged**